### PR TITLE
NOISSUE Rename 'walkable' tag to 'tile'

### PR DESCRIPTION
**Pull request by filip-szweda**
**lynx-scene-host-python:** https://github.com/group-project-gut/lynx-scene-host-python/pull/31
**lynx-frontend-godot:** https://github.com/group-project-gut/lynx-frontend-godot/pull/50
**lynx-scene-generator:** https://github.com/group-project-gut/lynx-scene-generator/pull/2

### CODE REVIEW

File: lynx/common/square.py [hunk 1]
```diff
@@ -1,38 +1,37 @@
...
 
 from lynx.common.object import Object
 
 
-# Class wraping logic of ground and objects placed on it
+# class wrapping logic of a tile and objects placed on it
 @dataclass
 class Square:
-    ground: Object = None
+    tile_object: Optional[Object] = None
```
Comment (EasyJB) [inline lynx/common/square.py:10]: I preferred ground name, but up to ur experience

**Review by EasyJB — APPROVED** [review]
LGTM

File: lynx/common/square.py (hunk 1 above)
Comment (Krystian030) [reply lynx/common/square.py:10]: I agree with @EasyJB

**Review by Krystian030 — APPROVED** [review]

File: lynx/common/square.py (hunk 1 above)
Comment (filip-szweda) [reply lynx/common/square.py:10]: "ground" is not telling what this one `Object` should always be (it should always be an `Object` with `tile` tag i.e. "tile_object")
Also, I would not want to introduce names that can be confused with "tile" or "square".

**Pull request merged**